esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -3,8 +3,6 @@
     UTC mode = True
 
 [task parameters]
-    # `fast` recipes are those taking less than 10 mins.
-    # Other recipes should be included in `medium`.
```

Comment:
Instead of removing these lines, would it be worth replacing them with the comment `See "How to add a recipe to the RTW" documentation for more information` or equivalent? 🤔

Reply:
Have added this as a replacement in commit d01a94f42